### DISCUSSION TITLE
Porter Publish from a Harbor to Azure ACR

### DISCUSSION THREAD

**Discussion by samwise-commits** [getporter/porter#2072] [Q&A] [answered]
So, I'm wondering if this is possible... or maybe there is a different way I can go about doing what I'm looking to do.

I have an already created porter bundle in my Harbor repository. I need to take that bundle and publish it to an Azure ACR. Typically to do this I make the tgz file and then do a porter publish archive on that file. Is there a way to publish the already published bundle (that is in harbor) to the Azure ACR? Or do I just need to create the file and publish it like I normally would? Basically trying to reduce pipeline time that would incur by having to do the full publish twice if I can just 'copy' the bundle from harbor to Azure ACR. Hopefully that makes sense. Thanks!

**Comment by samwise-commits** [comment, marked as answer]
Ah, well never mind! Didn't realize you already had a copy command! This works perfectly :)
https://porter.sh/copy-bundles/

**Reply by carolynvs** [reply]
I'm glad you got it figured out! 🎉
